content/en/flux/installation/upgrade.md

```diff
@@ -43,6 +43,15 @@ git add -A && git commit -m "Update $(flux -v) on my-cluster"
 git push
 ```
 
+If you've enabled the Flux [image automation feature](/flux/guides/image-update/) at bootstrap,
+make sure to include the image automation controllers when generating the components manifest:
```

Comment:
```suggestion
If you've enabled extra Flux components at bootstrap, like those required for the [image automation feature](/flux/guides/image-update/).
Ensure to include these components when generating the components manifest:
```

Makes it more future safe, in case we ever introduce new components.

In addition, maybe put this in some kind of warning / note / whatever block?

Reply:
There is an info box right after this, too many of those I think it just creates noise.